src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -166,10 +183,14 @@ public int getLevel() {
 
   @Override
   public String toString() {
-    return "UnitsDrawer for " + m_count + " " + MyFormatter.pluralize(m_unitType) + " in  " + m_territoryName;
+    return "UnitsDrawer for " + count + " " + MyFormatter.pluralize(unitType) + " in  " + territoryName;
   }
 
   private boolean isDamageFromBombingDoneToUnitsInsteadOfTerritories(final GameData data) {
     return games.strategy.triplea.Properties.getDamageFromBombingDoneToUnitsInsteadOfTerritories(data);
   }
+  
+  public static void setNationVisible(byte i){
```

Comment:
The static state is a code smell. We should be getting a props object injected into this class, but that is beyond the scope of this PR.

Regardless, this method would be helped with a renaming, something like:  `setUnitFlagDrawMode( UnitFlagDrawMode drawMode )`